pkg/core/resources/model/resource.go

```diff
@@ -158,6 +158,8 @@ type ResourceTypeDescriptor struct {
 	Insight Resource
 	// Overview contains the overview type attached to this resourceType
 	Overview Resource
+	// DumpForGlobal whether resources of this type should be dumped when exporting a zone to migrate to global
+	DumpForGlobal bool
```

Comment:
I'm not yet sure it will be simple on/off on a resource. I can imagine that for secrets/configs we might need to pick resources, but let's see.

Reply:
Ok we may simply drop this field then